### PR TITLE
New PR

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by henry-richard** [comment]
Github user commenting

File: src/foo.py [hunk 1]
```diff
@@ -8,4 +8,8 @@ def hello():
     print("Hello foo5")
     print("Hello foo6")
     print("Hello foo7")
+    print("Hello foo8")
```
Comment (henry-richard) [inline src/foo.py:11]: github user request review

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Comment by bert-griffin** [comment]
github user commentign

File: src/foo.py [hunk 2]
```diff
@@ -8,4 +8,8 @@ def hello():
     print("Hello foo5")
     print("Hello foo6")
     print("Hello foo7")
```
Comment (henry-richard) [inline src/foo.py:10]:
```suggestion
    print("Hello foo7 ")
```

**Review by henry-richard — APPROVED** [review]

**Review by henry-richard — APPROVED** [review]

**Pull request merged**